gnomad/assessment/validity_checks.py

```diff
@@ -857,12 +863,53 @@ def vcf_field_check(
     return True
 
 
+def compare_related_global_and_row_lengths(
```

Comment:
this minor, but I read this as applying only to the relatedness fields initially. what about "compare_relevant_global_and_row_lengths"? or "compare_global_and_row_annot_lengths"?

Reply:
Good point I think I favor "compare_global_and_row_annot_lengths"